include/deal.II/numerics/vector_tools.templates.h

```diff
@@ -917,8 +1050,8 @@ namespace VectorTools
       additional_data.mapping_update_flags = (update_values | update_JxW_values);
       MatrixFree<dim, Number> matrix_free;
       matrix_free.reinit (mapping, dof, constraints,
-                          QGauss<1>(fe_degree+2), additional_data);
```

Comment:
No, I thought you wanted me to leave the functions using `std_cxx11::function` as is, i.e. use `fe_degree+1`.
If I misunderstood you here, I happily make that consist with the original function again, i.e. use `fe_degree+2`.